qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy

```diff
@@ -227,7 +227,7 @@ class ProcessBaselinesTest extends BaseSpecification {
         assert (!StringUtils.isEmpty(lockProcessBaselines.get(0).getElements(0).getElement().processName))
 
         // sleep 5 seconds to allow for propagation to sensor
-        sleep 5000
+        sleep 10000
```

Comment:
[nit] there is another `sleep 5000` in line 246 in case we want to increase it too.

Reply:
done